### PR TITLE
update: update sentry example

### PR DESCRIPTION
**Pull request by taylrd**
Further clarify adding Sentry to the Epic Stack.

## Test Plan

Modified monitoring.md, deploy.yml and Dockerfile

## Checklist

- [ ] Tests updated
- [x ] Docs updated

### CODE REVIEW

File: docs/monitoring.md [hunk 1]
```diff
@@ -39,16 +39,15 @@ To generate the auth token, click
...
 to create an internal integration (which grants the selected capabilities to the
 recipient, similar to how RBAC works). Give it a name and add the scope for
-`Releases:Admin` and `Organization:Read`. Press Save, find the auth token at the
-bottom of the page under "Tokens", and copy that to secure location (this
-becomes `SENTRY_AUTH_TOKEN`). Then visit the organization settings page and copy
-that organization slug (`SENTRY_ORG`), and the slug name for your project too
+`Releases:Admin` and `Organization:Read`. Press Save, and then generate the auth token at the
+bottom of the page under "Tokens", and copy that to a secure location (this
+becomes `SENTRY_AUTH_TOKEN`). Then visit the organization general settings page and copy
+the organization slug (`SENTRY_ORG`), and the slug name for your project under Organization > Projects > Project > Name
```
Comment (kentcdodds) [inline docs/monitoring.md:45]: To ensure automatic formatting doesn't place a `>` at the beginning of a line.

```suggestion
the organization slug (`SENTRY_ORG`), and the slug name for your project under `Organization > Projects > Project > Name`
```

File: docs/monitoring.md [hunk 2]
```diff
@@ -39,16 +39,15 @@ To generate the auth token, click
...
 (`SENTRY_PROJECT`).
 
 In the 'build' section of the [Dockerfile](../other/Dockerfile), there is an
-example of how to pass (mount) the `SENTRY_AUTH_TOKEN` as a docker secret, so it
-is available to Vite when `npm run build` is run. You can do the same for
-`SENTRY_ORG` and `SENTRY_PROJECT` or actually any other secret (environment
+example of how to pass `SENTRY_AUTH_TOKEN`, `SENTRY_ORG` and `SENTRY_PROJECT` as secrets, so it
+is available to Vite when `npm run build` is run. Setup up your secrets in [GitHub Actions secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions).You can do the same for any other secret (environment
```
Comment (kentcdodds) [inline docs/monitoring.md:50]:
```suggestion
is available to Vite when `npm run build` is run. Setup up your secrets in [GitHub Actions secrets](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions). You can do the same for any other secret (environment
```

File: other/Dockerfile [hunk 3]
```diff
@@ -40,11 +48,7 @@ ADD prisma .
 RUN npx prisma generate
 
 ADD . .
-
-# Mount the secret and set it as an environment variable and run the build
-RUN --mount=type=secret,id=SENTRY_AUTH_TOKEN \
-    export SENTRY_AUTH_TOKEN=$(cat /run/secrets/SENTRY_AUTH_TOKEN) && \
-    npm run build
```
Comment (kentcdodds) [inline other/Dockerfile:47]: This should not be changed. Learn more about why it's accessed in this way from this PR: https://github.com/epicweb-dev/epic-stack/pull/660/files#diff-68a2c4544d4339258d27d291ed844a8fc6a79ad1b1cc7dcce28594c8d01c12beR45

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thanks! Just a few things.

File: other/Dockerfile (hunk 3 above)
Comment (taylrd) [reply other/Dockerfile:47]: Im not sure I can help then. When I add that back I'm back to the same errors

```
#17 30.85 [sentry-vite-plugin] Warning: No organization slug provided. Will not create release. Please set the `org` option to your Sentry organization slug.
#17 30.85 [sentry-vite-plugin] Warning: No org provided. Will not upload source maps. Please set the `org` option to your Sentry organization slug.
#17 30.93 [sentry-vite-plugin] Warning: No organization slug provided. Will not create release. Please set the `org` option to your Sentry organization slug.
#17 30.93 [sentry-vite-plugin] Warning: No org provided. Will not upload source maps. Please set the `org` option to your Sentry organization slug.
```

Comment (kentcdodds) [reply other/Dockerfile:47]: Did you make sure to set those values in GitHub actions as secrets?

I did the same to my own app just now and it worked well :)

https://github.com/kentcdodds/kentcdodds.com/commit/538080f2055a5090e904634f31259088dc8d603c

In my case, I just hard-coded the org and project though so maybe that's why?

Comment (taylrd) [reply other/Dockerfile]: I followed your commit and it works for me too. No errors and source maps generated successfully. I have updated this PR.

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Pull request merged**